sdk/python/feast/infra/online_stores/dynamodb.py

```diff
@@ -362,8 +377,12 @@ def from_proto(dynamodb_table_proto: DynamoDBTableProto) -> Any:
         )
 
     def update(self):
-        dynamodb_client = _initialize_dynamodb_client(region=self.region)
-        dynamodb_resource = _initialize_dynamodb_resource(region=self.region)
+        dynamodb_client = _initialize_dynamodb_client(
+            region=self.region, endpoint_url=None
+        )
+        dynamodb_resource = _initialize_dynamodb_resource(
+            region=self.region, endpoint_url=None
```

Comment:
Why is the endpoint_url None here?

Reply:
@achals my mistake, should we add it as an attribute to `DynamoDBTable` ?

Reply:
I'm not familiar with how the user interacts with the `DynamoDBTable` class, and how can the user configure it.

Reply:
Lets add it for consistency, it's not currently used right now but will be used as part of upcoming features.

Reply:
ok